operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -304,15 +283,11 @@ By default worker node personalization of iSCSI SBPS is done during CSM install/
 It is initiated during bootprep (`management-nodes-rollout`) in order to do worker node personalization
 automatically during boot time.
 
-#### Manual setup with CFS session
-
-Worker node personalization can be done post CSM install with CFS configuration session.
-Refer to [Node Personalization](../configuration_management/iSCSI_SBPS_Node_Personalization.md) for details.
+After the worker node personalization, verify the same using [Node Personalization Verification](Node_Personalization_Verification.md)
 
 ### 2. Run GOSS test suite
 
-In order to verify the readiness of the iSCSI targets before triggering the boot of compute nodes or UANs, it is important to run GOSS tests as
-sanity checks on iSCSI targets.
+In order to verify the readiness of the iSCSI targets before triggering the boot of compute nodes or UANs, it is important to run GOSS tests as sanity checks on iSCSI targets.
 
 Refer to [GOSS tests for SBPS](https://github.com/Cray-HPE/sbps-marshal/blob/main/GOSS_tests_for_sbps.md) for the details.
```

Comment:
This same link is used also in the CSM 1.6 docs. But changes were made to some of these things between CSM 1.6 and CSM 1.7. And looking at the target page, it mostly seems to be intended for CSM 1.6 (the test scripts it links to are from the `csm-testing` CSM 1.6 branch, for example).

This link needs to point to relevant information for CSM 1.7.

The ideal solution would be to move the content from that other repo into `docs-csm`. Otherwise, some method will need to be used to make sure that `docs-csm` always points to the appropriate version of that page, even as changes are made to that page over new releases of SBPS.

Reply:
I don't see any changes that have been made to address this. Can you point out the changes you made that you believe have addressed it?

Reply:
What text changes? I didn't make any specific suggestions for text changes.